docs/current_architecture.md

```diff
@@ -23,18 +25,22 @@ recommendationservice(Recommendation Service):::python
 shippingservice(Shipping Service):::rust
 featureflagservice(Feature Flag Service):::erlang
 featureflagstore[(Feature Flag Store<br/>&#40PostgreSQL DB&#41)]
+queue[(queue<br/>&#40Kafka&#41)]
 
 Internet -->|HTTP| frontendproxy
 frontendproxy -->|HTTP| frontend
 frontendproxy -->|HTTP| featureflagservice
 loadgenerator -->|HTTP| frontend
 
+accounting -->|TCP| queue
+
```

Comment:
1. Should the name be accountingservice vs. accounting (as other services have service suffix at end). I am assuming the diagram has space for this. 
2. As accountingservice is consumer maybe the arrow should be reversed (going from queue to accountingservice).
Same suggestion for fraud detection service.